io/module_wrt_grid_comp.F90

```diff
@@ -65,29 +65,29 @@ module module_wrt_grid_comp
       integer,save      :: ntasks                                         !< Number of write tasks in the current group.
       integer,save      :: itasks                                         !< Number of write tasks in i direction in the current group.
       integer,save      :: jtasks                                         !< Number of write tasks in j direction in the current group.
-      integer,save      :: ngrids                                         !< ???
+      integer,save      :: ngrids                                         !< Number of output grids
 
       integer,save      :: wrt_mpi_comm                                   !< The mpi communicator in the write comp.
-      integer,save      :: idate(7)                                       !< ???
-      integer,save      :: start_time(7)                                  !< ???
-      logical,save      :: write_nsflip                                   !< ???
-      logical,save      :: change_wrtidate=.false.                        !< ???
-      integer,save      :: frestart(999) = -1                             !< ???
-      integer,save      :: calendar_type = 3                              !< ???
-      logical           :: lprnt                                          !< ???
+      integer,save      :: idate(7)                                       !< IO base time array
+      integer,save      :: start_time(7)                                  !< Experiment starting time
+      logical,save      :: write_nsflip                                   !< Flag to flip vertical coordinate
```

Comment:
I thought this flag controls the flipping of y-coordinate (north-south), which is not 'vertical'.

Reply:
Updated